cluster/gce/gci/configure.sh

```diff
@@ -236,10 +236,10 @@ function install-node-problem-detector {
 
 function install-cni-binaries {
   if [[ -n "${CNI_VERSION:-}" ]]; then
-      local -r cni_tar="cni-plugins-amd64-${CNI_VERSION}.tgz"
+      local -r cni_tar="cni-plugins-linux-amd64-${CNI_VERSION}.tgz"
       local -r cni_sha1="${CNI_SHA1}"
```

Comment:
We should also compute "${CNI_STORAGE_PATH}/${CNI_VERSION}/${cni_tar}" to a local in each of these branches and then use that below.